core/tests/transaction.rs

```diff
@@ -28,13 +30,15 @@ fn transaction_parse() -> Result<(), Error> {
     let outputs = [note.clone(), note];
     let tx_max_fee = 0;
     let deposit = 0;
+    let recipient_params = RecipientParameters::default();
```

Comment:
I would like to test the serialization with values other than only zeros.
```suggestion
    let recipient_params = RecipientParameters::new(
        &mut rng,
        &sk,
        [
        note.stealth_address().note_pk().as_ref().into(),
        note.stealth_address().note_pk().as_ref().into()
        ],
        BlsScalar::from(42);
    );
```
you need to see whether the note_pk can be constructed like this, you might need to use `JubJubAffine::from` instead of `into`.